From 257983a046b93ac082546577a227461239e14343 Mon Sep 17 00:00:00 2001
From: Andy Ross <andrew.j.ross@intel.com>
Date: Sat, 7 Jun 2014 14:09:57 -0700
Subject: [PATCH 066/117] gc0339: fix module name

Userspace uses this name to choose a CPF file, and this one doesn't
match the tuning we were given.  It's non-standard anyway; barring
strange situations this should always match the module name.

Signed-off-by: Andy Ross <andrew.j.ross@intel.com>
---
 drivers/external_drivers/camera/drivers/media/i2c/gc0339.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/external_drivers/camera/drivers/media/i2c/gc0339.h b/drivers/external_drivers/camera/drivers/media/i2c/gc0339.h
index b04ab99..986cae1 100644
--- a/drivers/external_drivers/camera/drivers/media/i2c/gc0339.h
+++ b/drivers/external_drivers/camera/drivers/media/i2c/gc0339.h
@@ -35,7 +35,7 @@
 
 #include <linux/atomisp_platform.h>
 
-#define GC0339_NAME		"gc0339_raw"
+#define GC0339_NAME		"gc0339"
 
 /* Defines for register writes and register array processing */
 #define I2C_MSG_LENGTH		1
-- 
1.9.3

